<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bopmgWindow</class>
 <widget class="QWidget" name="bopmgWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BOP-MG</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_connection">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_status">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string>not connected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cob_ports"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_resetRefresh">
        <property name="text">
         <string>reset / refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_clearInfo">
        <property name="text">
         <string>clear Info</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chb_setZeroAtStopSignal">
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="text">
      <string>set 0 at stop signal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_emergencyStop">
     <property name="text">
      <string>EMERGENCY EXIT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_values">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_setValues">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_setValue_lbl">
           <property name="text">
            <string>set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cob_setValue"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lbl_setTo_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>to</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="dsp_setValue">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="cob_setValueUnit"/>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="btn_setValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>set</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QCheckBox" name="chb_adjustSetValue">
           <property name="text">
            <string>adjust value</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="QLabel" name="lbl_setValueSet">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;not set (measure before seting power)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_voltage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_voltage_lbl">
           <property name="text">
            <string>voltage:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_voltage">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_shareVoltage">
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_current">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_current_lbl">
           <property name="text">
            <string>current:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_current">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_shareCurrent">
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_power">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_power_lbl">
           <property name="text">
            <string>power:  </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_power">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_sharePower">
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="frame_resistance">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_resistance_lbl">
           <property name="text">
            <string>resistance:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_resistance">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_shareResistance">
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QFrame" name="frame_measuredValues">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_measuredValues_lbl">
           <property name="text">
            <string>measured values:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cob_measuredValues"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_measuredValuesVisibility">
           <property name="text">
            <string>hide</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
